### PR TITLE
create AvatarGroup reusable components

### PR DESCRIPTION
**Pull request by lucaslyl**
1) Done extend the avatar with new thumbnail args,
2) Intergrate Avatar boxel-ui  components to avatar group

Avatar UI component:
image

[RESULT] Avatar Group with thumbnail:
Screenshot 2024-12-04 at 07 21 25

[RESULT] Avatar Group without thumbnail:
Screenshot 2024-12-04 at 07 22 07

### CODE REVIEW

File: packages/experiments-realm/components/avatar-group.gts [hunk 1]
```diff
@@ -0,0 +1,79 @@
+import GlimmerComponent from '@glimmer/component';
+import { htmlSafe } from '@ember/template';
+import UserCircleIcon from '@cardstack/boxel-icons/user-circle';
+
+interface AvatarGroupSignature {
+  Args: {
+    thumbnailURL?: string;
+    name?: string;
+  };
+  Blocks: { company: [] };
```
Comment (tintinthong) [inline packages/experiments-realm/components/avatar-group.gts]: call this content to be more generic

File: packages/experiments-realm/components/avatar-group.gts [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  Element: HTMLElement;
+}
+
+export default class AvatarGroup extends GlimmerComponent<AvatarGroupSignature> {
```
Comment (tintinthong) [inline packages/experiments-realm/components/avatar-group.gts:15]: Where is this component used?

File: packages/experiments-realm/components/avatar-group.gts (hunk 1 above)
Comment (lucaslyl) [reply packages/experiments-realm/components/avatar-group.gts]: done

File: packages/boxel-ui/addon/src/helpers/deterministic-color-from-string.ts [hunk 3]
```diff
@@ -9,7 +9,7 @@ import {
 // Selects a random color from a set of colors based on the input string
 export function deterministicColorFromString(str: string): string {
   if (!str) {
-    return 'transparent';
+    return '#EEEEEE';
```
Comment (lucaslyl) [inline packages/boxel-ui/addon/src/helpers/deterministic-color-from-string.ts:12]: set the default color if the user didn't pass in the str value

File: packages/experiments-realm/components/avatar-group.gts [hunk 4]
```diff
@@ -0,0 +1,67 @@
+import GlimmerComponent from '@glimmer/component';
+import { htmlSafe } from '@ember/template';
+import { Avatar } from '@cardstack/boxel-ui/components';
+
+interface AvatarGroupSignature {
+  Args: {
+    thumbnailURL?: string;
+    name: string;
```
Comment (tintinthong) [inline packages/experiments-realm/components/avatar-group.gts]: name here is possibly undefined

**Review by tintinthong — APPROVED** [review]

**Pull request merged**